CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+## Contributing Guidelines
```

Comment:
This file is used for ReadTheDocs site kpp.readthedocs.io.  We can switch that over from geoschem/KPP to Kinetic PreProcessor/KPP.